src/app/lib/config/services/loadableConfig.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * This config is used by `createLoadableContext`
+ * to dynamically load each service's config.
+ */
+const loadableConfig = {
+  news: {
+    loader: () => import(/* webpackChunkName: 'news' */ './news.js'),
+    webpack: () => [require.resolveWeak('./news.js')],
```

Comment:
**Important comment:**
This webpack key is set to allow Loadable.preloadReady to work. Loadable determines which modules are loaded using references found using this key. More info can be found [here](https://github.com/jamiebuilds/react-loadable#declaring-which-modules-are-being-loaded). The reason we need to set it explictly, rather than using `react-loadable/babel` is due to the way we dynamically create loadable components, as the babel parser cant find them all as it doesnt run the code.